app/components/App.js

```diff
@@ -1,9 +0,0 @@
-import React from 'react';
-
-const App = () => (
-  <div>
-    Internship Portal
-  </div>
-);
```

Comment:
It is better not to remove App.js since it will be required for react-router. You can use this:
```
const App = () => (
  <div>
    <Navbar />
    <PostView />
  </div>
);
```

Reply:
ok then,  i am merging the changes.

Reply:
Why? We can get the configured routes from `configureRoutes.js` file and use that in the `client.js` file. Something along the line.
```javascript
ReactDOM.render(
    <BrowserRouter>
        {routes}
    <BrowserRouter>,
    document.getElementById('app'),
);
```
I think this will be a better approach considering the fact that we would be using `redux` along the way. We can discuss over it later.